src/App.css

```diff
@@ -0,0 +1,312 @@
+  justify-content: space-between;
+}
+
+.circle-wrap {
+  margin: 5px 0;
+  width: 150px;
+  height: 150px;
+  background: #fafafa;
+  border-radius: 50%;
+  border: 1px solid #fafafa;
+  transform: scale(0.6);
+}
+
+.mask .fill {
+  clip: rect(0, 75px, 150px, 0);
+  background-color: #227ded;
+}
+
+.mask.full,
+.circle .fill {
+  animation: fill ease-in-out 3s;
+  transform: rotate(135deg);
+}
+
+.circle-wrap .circle .mask,
+.circle-wrap .circle .fill {
+  width: 150px;
+  height: 150px;
+  position: absolute;
+  border-radius: 50%;
+}
+
+.circle-wrap .circle .mask {
+  clip: rect(0, 150px, 150px, 75px);
+}
+
+.circle-wrap .inside-circle {
+  width: 122px;
+  height: 122px;
+  border-radius: 50%;
+  background: #fff;
+  line-height: 120px;
+  text-align: center;
+  margin-top: 14px;
+  margin-left: 14px;
+  color: #1e51dc;
+  position: absolute;
+  z-index: 100;
+  font-weight: 700;
+  font-size: 2em;
+}
+
+@keyframes fill {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(135deg);
+  }
+}
+
+.second {
+  display: flex;
+  align-items: center;
+}
+
+.percentage {
+  margin: 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
+.completed {
+  margin: 0;
+  opacity: 0.5;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.third {
+  margin: 0 40px 0 0;
+}
+
+.current {
+  opacity: 0.5;
+  margin: 1.5rem 0 0 0;
+  font-family: 'Roboto Slab', sans-serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.chapter {
+  margin: 5px 0 25px 0;
+  font-family: 'Roboto Slab', sans-serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
+
+.progress-btn {
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border: none;
+}
+
+.progress {
+  font-family: 'Roboto Slab', sans-serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+}
+
+.main-add {
+  margin: 0 40px 20px 40px;
+  width: 90%;
+}
+
+.form {
+  width: 97.5%;
+  display: flex;
+  align-items: center;
+  margin: 0 0 20px 0;
+}
+
+.title-form {
+  width: 42%;
+  height: 1.8rem;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.7rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.15px;
+  color: #121212;
+  padding: 0 0.8rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+}
```

Comment:
- [ ] Kindly refer to the first comment for the color `#e8e8e8` used many times. 👍